src/style.css

```diff
@@ -0,0 +1,88 @@
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  font-style: normal;
+}
+
+body {
+  --border-color: #e1e7ed;
+  --input-color: #3d4e61;
```

Comment:
Move to `root` selector

[More info](https://developer.mozilla.org/en-US/docs/Web/CSS/:root)